lib/src/crypto_subtle.dart

```diff
@@ -73,22 +42,142 @@ Uint8List bigIntToUint8ListBigInteger(BigInt integer) {
+    JSAny algorithm,
+    JSAny key,
+    JSTypedArray data,
+  );
 
-  CryptoKey(this._jsObj);
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/sign
+  external JSPromise<JSArrayBuffer> sign(
+    JSAny algorithm,
+    JSCryptoKey key,
+    JSTypedArray data,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/verify
+  external JSPromise<JSBoolean> verify(
+    JSAny algorithm,
+    JSCryptoKey key,
+    JSTypedArray signature,
+    JSTypedArray data,
+  );
 
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest
+  external JSPromise<JSArrayBuffer> digest(
+    String algorithm,
+    JSTypedArray data,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/deriveBits
+  external JSPromise<JSArrayBuffer> deriveBits(
+    JSAny algorithm,
+    JSCryptoKey baseKey,
+    int length,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/generateKey
+  @JS('generateKey')
+  external JSPromise<JSCryptoKey> generateCreptoKey(
+    JSAny algorithm,
+    bool extractable,
+    JSArray<JSString> keyUsages,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/generateKey
+  @JS('generateKey')
+  external JSPromise<JSCryptoKeyPair> generateCreptoKeyPair(
+    JSAny algorithm,
+    bool extractable,
+    JSArray<JSString> keyUsages,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/importKey
+  @JS('importKey')
+  external JSPromise<JSCryptoKey> importKey(
+    String format,
+    JSTypedArray keyData,
+    JSAny algorithm,
+    bool extractable,
+    JSArray<JSString> keyUsages,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/importKey
+  @JS('importKey')
+  external JSPromise<JSCryptoKey> importJsonWebKey(
+    String format,
+    JSAny keyData,
+    JSAny algorithm,
+    bool extractable,
+    JSArray<JSString> keyUsages,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/exportKey
+  @JS('exportKey')
+  external JSPromise<JSArrayBuffer> exportKey(
+    String format,
+    JSCryptoKey key,
+  );
+
+  /// https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/exportKey
+  @JS('exportKey')
+  external JSPromise<JSJsonWebKey> exportJsonWebKey(
+    String format,
+    JSCryptoKey key,
+  );
+}
+
+/// Wrapper for the [CryptoKey][1] type.
+///
+/// [1]: https://www.w3.org/TR/WebCryptoAPI/#cryptokey-interface
+/// https://developer.mozilla.org/en-US/docs/Web/API/CryptoKey
+extension type JSCryptoKey(JSObject _) implements JSObject {
   /// Returns the _type_ of this key, as one of:
   ///  * `'private'`
   ///  * `'public'`
   ///  * `'secret'`
-  String get type => getProperty(_jsObj, 'type');
+  /// https://developer.mozilla.org/en-US/docs/Web/API/CryptoKey/type
+  external String get type;
 
   /// True, if this key can be extracted.
-  bool get extractable => getProperty(_jsObj, 'extractable');
+
```

Comment:
```suggestion
///
```
missing empty documentation comment line?

Reply:
I added empty lines. [55af27e](https://github.com/google/webcrypto.dart/pull/86/commits/55af27e9403b77c5437f947ee9158f7b7805838a)